src/parsepkg.h

```diff
@@ -99,6 +100,7 @@ struct cr_XmlStruct cr_xml_from_rpm(const char *filename,
                                     const char *location_href,
                                     const char *location_base,
                                     int changelog_limit,
+                                    gboolean filelists_ext,
```

Comment:
Could you create a new function instead of changing this one?

Reply:
yes, done, but in the Python side I kept one, as it is an optional parameter that does not break the old signature function